.travis.yml

```diff
@@ -14,7 +14,7 @@ git:
 # command to run scripts
 script:
   - pylint mlfinlab --rcfile=.pylintrc -f text
-  - python -m unittest discover
+  - "travis_wait 20 sleep 1200 &"
```

Comment:
Re our call, we need to correct this.

Reply:
Done. Alternative methods didn't work, so I'm keeping the version without `travis_wait` and decreasing verbose outputs to console.